skyportal/handlers/api/photometry.py

```diff
@@ -261,6 +264,8 @@ def put(self, photometry_id):
 
         phot.original_user_data = packet
         phot.id = photometry_id
+        if group_ids is not None:
```

Comment:
When we every want to upload photometry if there are no `group_ids` associated with it?

Reply:
This is the `put` handler, not the `post` handler.